sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -53,6 +53,7 @@ public partial class ProcessSessionMessageEventArgs : System.EventArgs
         public virtual System.Threading.Tasks.Task DeadLetterMessageAsync(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message, string deadLetterReason, string deadLetterErrorDescription = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task DeferMessageAsync(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message, System.Collections.Generic.IDictionary<string, object> propertiesToModify = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<System.BinaryData> GetSessionStateAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual void MarkSessionClosed() { }
```

Comment:
May I suggest `RequestSessionClose` for consideration?

Reply:
That seems reasonable, I want to let the architects weigh in.
/cc @KrzysztofCwalina @tg-msft

Reply:
I don't really like `Mark` here and try to avoid the words `Request`/`Response`/`Pipeline`/etc. since they're so heavily used today.  What about `AbandonSession`/`SurrenderSession`/`RelinquishSession`/`LeaveSession`/`VacateSession`/`ResignSession`/`YieldSession` since it continues to exist but we're done using it?

Reply:
I see it as being more about that the action is not occurring right now than whether the session continues to exist. Even with AbandonSession, it makes it seem like the session is immediately abandoned, when it really isn't. This is true in particular of Abandon, because there is a service operation on messages with the same name. For the others, you could argue, that the "not closing it immediately" is implied by the name. Still I think we should probably include Close in the name since we have the SessionClosingAsync event. 

What about `InitiateSessionClose'?

Reply:
What about `ReleaseSession` then?  That's a well known verb for saying "I'm done with it, but don't delete it until everyone else is too".

Reply:
Good idea